config-graphql.js

```diff
@@ -0,0 +1,53 @@
+const namingConventionRule = {
+  InputValueDefinition: { style: 'camelCase' },
+  InputObjectTypeDefinition: { style: 'PascalCase', requiredSuffixes: ['Input'] },
+  FieldDefinition: 'camelCase',
+  'FieldDefinition[parent.name.value=Mutation]': { style: 'camelCase', forbiddenSuffixes: ['Mutation'] },
+  'FieldDefinition[parent.name.value=Query]': { style: 'camelCase', forbiddenSuffixes: ['Query'] },
+  allowLeadingUnderscore: true,
```

Comment:
Would also love to see some forbidden prefixes here, like `get` and `upsert`, is that possible?

Reply:
yup,it is. I would like to first make clear to everyone (engineering team) that we are not supposed to be using **get** and **upsert** because I think that's not a common knowledge.  

https://the-guild.dev/graphql/eslint/rules/naming-convention#forbiddenprefixes-array